spec/factories/users.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :user do
+    name { "MyString" }
```

Comment:
I'm not a pro at Factory Bot, so just genuinely curious whether it is best practice to leave the "MyString" or replace it with something like` {  Faker::Name.first_name }` ?

Reply:
Believe it or not, this is actually autogenerated by Factorybot! We haven't changed it all, but adding Faker in is probably a good idea

Reply:
I think for now it is ok. If there is a situation where we need to create multiple records of a user, then we can implement Faker for these fields

Reply:
I prefer to use faker with factory bot, like Carson was saying, but we can worry about that later.